### DISCUSSION TITLE
开启openclash以后ipv6就失效了怎么办~

### DISCUSSION THREAD

**Discussion by Pacio** [vernesong/OpenClash#3892] [Q&A]
移动宽带，ipv6还是挺重要的，我自己编译固件一般都是开启ipv6协议的~
搜索了issue和讨论没发现有效的解决方案啊

**Comment by LemonPAKA** [comment]
> 移动宽带，ipv6还是挺重要的，我自己编译固件一般都是开启ipv6协议的~ 搜索了issue和讨论没发现有效的解决方案啊

openclash里开启了允许ipv6了没有，另外就是是不是使用了fake-ip，用meta改为redir-host试一下

**Comment by Pacio** [comment]
> > 移动宽带，ipv6还是挺重要的，我自己编译固件一般都是开启ipv6协议的~ 搜索了issue和讨论没发现有效的解决方案啊
> 
> openclash里开启了允许ipv6了没有，另外就是是不是使用了fake-ip，用meta改为redir-host试一下
允许了，现在是fake-ip混合模式，redit-host以前我用过，ipv6也一样不通，我目前访问ipw.cn是可以检测到ipv6优先的，但是访问www.test-ipv6.com检测到的是vps的ipv4地址，感觉是分流问题

**Comment by LemonPAKA** [comment]
> > > 移动宽带，ipv6还是挺重要的，我自己编译固件一般都是开启ipv6协议的~ 搜索了issue和讨论没发现有效的解决方案啊
> > 
> > 
> > openclash里开启了允许ipv6了没有，另外就是是不是使用了fake-ip，用meta改为redir-host试一下
> > 允许了，现在是fake-ip混合模式，redit-host以前我用过，ipv6也一样不通，我目前访问ipw.cn是可以检测到ipv6优先的，但是访问www.test-ipv6.com检测到的是vps的ipv4地址，感觉是分流问题

fakeip的话ipw.cn返回的是你路由自身的IPV6（但是应该能V6优先），如果开了绕开大陆的话，返回的应该是你机子的V6？
另外不是V6优先的话可能是DNS的问题
